src/main/webapp/app/core/home/home.component.ts

```diff
@@ -159,6 +214,39 @@ export class HomeComponent implements OnInit, AfterViewChecked {
         }
     }
 
+    /**
+     * Checks if the error is related to passkey autocomplete and makes sure to reEnable it if necessary.
+     *
+     * @param error that occurred during the passkey login process.
+     * @return true if the error is an expected error related to making passkey autocomplete available
+     */
+    private isPasskeyAutocompleteError(error: Error): boolean {
+        if (error.name === USER_CANCELLED_LOGIN_WITH_PASSKEY_ERROR) {
+            // The user manually aborted the passkey login process after clicking the "Sign in with passkey" button.
+            // This required aborting the `getCredential` request with conditional mediation, which is necessary for enabling passkey autocomplete.
+            // To restore passkey autocomplete functionality, re-invoke `makePasskeyAutocompleteAvailable`.
+            // eslint-disable-next-line no-undef
+            console.warn('Operation not allowed or timed out: login with passkey was aborted manually by the user');
```

Comment:
Is this intentionally here, or was it for testing?

Reply:
This is intentional, but "by the user" sounds a little odd if you read it in the console - I adjusted it 👍

Reply:
While I do think that it is good practice to use browser features, it also comes with the issue that Browser also contain bugs and do things differently, especially regarding the new introduction of passkey features I do think it is good to log things to the console for now as many environment specific settings can lead to bugs (and exact errors and warnings help us reproduce these issues)